src/components/settings/General.vue

```diff
@@ -37,6 +37,42 @@
                 </select>
             </div>
 
+            <!-- Time Format -->
+            <div class="mb-4">
+                <label class="form-label">
+                    {{ $t("Time Format") }}
+                </label>
+
+                <div class="form-check">
+                    <input
```

Comment:
we should add a component for this (not to be done in this PR) ^^

Reply:
Agreed. Happy to raise another PR for this.